lib/views/splash_view.dart

```diff
@@ -1,4 +1,8 @@
+import 'dart:io';
```

Comment:
there's a lot of logic here that shouldn't be in the view. Remember it's good to keep the views as dumb as possible

Reply:
Alright, I'll take care of it.